server/base/src/main/java/org/apache/accumulo/server/ServerOpts.java

```diff
@@ -20,6 +20,7 @@
 
 import com.beust.jcommander.Parameter;
 
+@Deprecated
 public class ServerOpts extends ConfigOpts {
 
   @Parameter(names = {"-a", "--address"}, description = "address to bind to")
```

Comment:
This could be deleted.  It is not being used unless making this a deprecated option is desired.

Reply:
This could be replaced with a simple check of the args list for any `-a` or `--address` strings, print an error message with the new replacement option. Could also permit this, with a warning, and use its value to override whatever the configuration parsing results in... but that would add some complexity that might not be worth it (as you've already alluded to in your PR comment body).